app/views/devise_otp/credentials/refresh.html.erb

```diff
@@ -9,12 +9,12 @@
     <%= f.text_field :email, :disabled => :true%></div>
 
     <div><%= f.label :password %><br />
-	<%= f.password_field :refresh_password, :autocomplete => :off, :autofocus => true %></div>
+  <%= f.password_field :refresh_password, :autocomplete => :off, :autofocus => true %></div>
```

Comment:
This is a 3rd party library though, we should change as little as possible.  Think about what kind of pull request you would submit to the upstream project -- they would not want whitespace changes mixed into an unrelated pull request.